taipy/core/submission/_submission_manager.py

```diff
@@ -53,3 +54,26 @@ def _get_latest(cls, entity: Union[Scenario, Sequence, Task]) -> Optional[Submis
             return submissions_of_task[0]
         else:
             return max(submissions_of_task)
+
+    @classmethod
+    def _is_editable(cls, entity: Union[Submission, str]) -> bool:
+        return False
+
+    @classmethod
+    def _delete(cls, submission: Union[Submission, SubmissionId]):
+        if isinstance(submission, str):
+            submission = cls._get(submission)
+        if cls._is_deletable(submission):
+            super()._delete(submission.id)
+        else:
+            err = SubmissionNotDeletedException(submission.id)
+            cls._logger.warning(err)
```

Comment:
Warning and not exception?

Reply:
Does it make more sense to be an error instead? @jrobinAV what do you think?

Reply:
I'll keep it as error log then